<launch>
    <!-- -->
    <!-- image publisher -->
    <node name="image_file_publisher" pkg="nodelet" type="nodelet" 
          args="standalone image_source/ImageFiles" output="screen">
        <remap from="image_out" to="image"/>
        <param name="loop" value="true"/>
        <param name="recursive" value="true"/>
        <param name="publish_by_call" value="true"/>
        <!-- <param name="rate" value="2.0"/> -->
        <rosparam param="files" subst_value="true">
            - [ "dir1", ".+\\.jpg", "bgr8" ]
            - [ "dir2", ".+\\.png", "mono8" ]
        </rosparam>
    </node>
    <!-- -->
    <!-- image viewer -->
    <node name="image_viewer" pkg="nodelet" type="nodelet"
          args="standalone image_view/image" output="screen">
        <param name="image_transport" value="compressed"/>
    </node>
</launch>